assets/bingo/lang/de_de.json

```
@@ -1,17 +1,286 @@
+	"bingo.lobby.tutorial.game_info.world_gen.ites.heading": "Keine *ite: ",
+	"bingo.lobby.tutorial.game_info.world_gen.ites": "Alle Steinvarianten wie DiorIT, GranIT and AndesIT sind in dieser Version komplett entfernt.\nDadurch soll das Zumüllen des Inventars reduziert werden.",
+	"bingo.lobby.tutorial.game_info.world_gen.ambient.heading": "Umgebungsbeleuchtung: ",
+	"bingo.lobby.tutorial.game_info.world_gen.ambient": "Du kannst im Dunkeln besser sehen, aber immer noch hell von dunkel unterscheiden.",
+
+	"bingo.lobby.tutorial.get_tools.sign.line1": "Nach dem Start",
+	"bingo.lobby.tutorial.get_tools.sign.line2": "brauchst Du ",
+	"bingo.lobby.tutorial.get_tools.sign.line3": "erstmal ",
+	"bingo.lobby.tutorial.get_tools.sign.line4": "Werkzeug.",
+
+	"bingo.lobby.team_selection.sign.line1": "",
+	"bingo.lobby.team_selection.sign.line2": "Team",
+	"bingo.lobby.team_selection.sign.line3": "wechseln",
+	"bingo.lobby.team_selection.sign.line4": "",
+
+	"bingo.lobby.end.sign.line1": "Die Außenwelt",
+	"bingo.lobby.end.sign.line2": "",
+	"bingo.lobby.end.sign.line3": "Gefahr!",
+	"bingo.lobby.end.sign.line4": "Bleibt drinnen!",
+
+	"bingo.categories.header": "Kategorien:",
+	"bingo.category.apple": "Apfel",
+	"bingo.category.bow": "Bogen",
+	"bingo.category.bricks": "Ziegel",
+	"bingo.category.cactus": "Kaktus",
+	"bingo.category.cactus_bone": "Kaktus und Knochen",
+	"bingo.category.chest_iron": "Eisen und Truhen",
+	"bingo.category.chest_loot": "Verlies Beute",
+	"bingo.category.compass": "Kompass",
+	"bingo.category.diamond": "Diamant",
+	"bingo.category.egg": "Ei",
+	"bingo.category.nighttime_mob_drops": "Nacht Mobs Beute",
+	"bingo.category.extra_smelt": "Extra Ofen Zeit",
+	"bingo.category.fish": "Fisch",
+	"bingo.category.flint": "Feuerstein",
+	"bingo.category.furnace": "Ofen",
+	"bingo.category.gold_and_ruined_portal": "Items von Portalruinen oder aus Gold",
+	"bingo.category.gunpowder": "Schwarzpulver",
+	"bingo.category.ink": "Tinte",
+	"bingo.category.ink_bone": "Tinte und Knochen",
+	"bingo.category.jungle": "Dschungel",
+	"bingo.category.rabbit": "Kaninchen",
+	"bingo.category.lapis": "Lapis Lazuli",
+	"bingo.category.leather": "Leder",
+	"bingo.category.lime": "Hellgrün",
+	"bingo.category.milk": "Milch",
+	"bingo.category.mushroom": "Pilze",
+	"bingo.category.pumpkin": "Kürbis",
+	"bingo.category.rail": "Schienen",
+	"bingo.category.redstone": "Einfacher Redstone",
+	"bingo.category.sand": "Sand",
+	"bingo.category.sapling": "Setzlinge",
+	"bingo.category.shearable": "Scheerbares",
+	"bingo.category.snow": "Schnee",
+	"bingo.category.spider": "Spinnenauge",
+	"bingo.category.taiga": "Taiga",
+	"bingo.category.treasure_chest": "Übliche Beute aus vergrabenen Schätzen",
+	"bingo.category.wheat": "Weizen",
+	"bingo.category.wool": "Wolle",
+	"bingo.items.weight": "Gewicht: %s",
```

```suggestion
	"bingo.items.weight": "Gewichtung: %s",
```
Weight in probability is "Gewichtung" in German, not "Gewicht"